src/roman_datamodels/util.py

```diff
@@ -2,27 +2,28 @@
 Various utility functions and data types
 """
 
+import logging
 import os
-from platform import system as platform_system
-import psutil
 import traceback
+from platform import system as platform_system
 from pydoc import locate
 
-#from . import s3_utils
-#from .basic_utils import bytes2human
+import psutil
+
+# from . import s3_utils
+# from .basic_utils import bytes2human
```

Comment:
Should these lines be deleted?

Reply:
Sure, I can do that. There are some more extreme `ruff` checks that will autofix and/or flag this stuff.